ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -121,7 +121,7 @@ def ui_add_capacity_conditions():
     ):
         logger.info(f"Add capacity via UI is not supported on platform {platform}")
         return False
-    elif ocp_version not in ("4.7", "4.8", "4.9", "4.10", "4.11", "4.12"):
+    elif ocp_version not in ("4.7", "4.8", "4.9", "4.10", "4.11", "4.12", "4.13"):
```

Comment:
I'm not sure, if we should keep the `4.13` related changes in the backport for `release-4.12` branch. @petr-balogh, @clacroix12, @vavuthu  WDYT?

Reply:
done